playbooks/roles/ocp-config/tasks/restart-config.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# Add restart always configurations to the services
+- name: Create squid dropin directory
```

Comment:
IMO this task should be moved to `squid.yaml` file since we already have a squid specific file.